modin/config/envvars.py

```diff
@@ -621,26 +703,82 @@ class GithubCI(EnvironmentVariable, type=bool):
     default = False
 
 
-class ExperimentalNumPyAPI(EnvironmentVariable, type=bool):
-    """Set to true to use Modin's experimental NumPy API."""
+class NumpyOnModin(EnvWithSibilings, type=bool):
```

Comment:
```suggestion
class NumPyAPI(EnvWithSibilings, type=bool):
```
NumpyOnModin seems a little vague to me. We think of an API implementation in Modin as the API on an engine. I would suggest to use just `NumPyAPI`.

Reply:
`NumPyAPI` seems even more vague to me. Maybe something like `NumpyImpl` or idk `NumpyAPIOnModin`?

> We think of an API implementation in Modin as the API on an engine

Going from this description `NumpyOnModin` fits the best IMHO. The name doesn't mention any engine that's being used for this implementation, it's just indicating that numpy-calls are now being run via Modin using whatever engine is currently selected

Reply:
Okay, `NumpyOnModin` is fine by me. Just noticed that we don't still have any section in docs regarding NumPy API in Modin. @RehanSD, as far as I remember, you wanted to add something on this matter. Any plans?

Reply:
Or we could use ModinNumpy config and MODIN_NUMPY env var for consistency.

Reply:
> Or we could use ModinNumpy config and MODIN_NUMPY env var for consistency.

renamed